Cis partitions rules

willumpie opened the pull request:
#### Description:

- Add partitions rules for the CIS profile for Ubuntu 1804

#### Rationale:
Make rules available for Ubuntu 1804

pep8speaks commented:
Hello @willumpie! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:











* In the file [`utils/upload_ansible_roles_to_galaxy.py`](https://github.com/ComplianceAsCode/content/blob/8bc332b8a50204ba03c783f4108ae6e39150d73e/utils/upload_ansible_roles_to_galaxy.py):

> [Line 504:100](https://github.com/ComplianceAsCode/content/blob/8bc332b8a50204ba03c783f4108ae6e39150d73e/utils/upload_ansible_roles_to_galaxy.py#L504): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (126 > 99 characters)

redhatrises commented:
@willumpie can do a rebase please? We would like to not have all the extra commits in this PR. Makes it very hard to review.

willumpie commented:
> @willumpie can do a rebase please? We would like to not have all the extra commits in this PR. Makes it very hard to review.
Of course.

willumpie commented:
> @redhatrises  can do a rebase please? We would like to not have all the extra commits in this PR. Makes it very hard to review.

I merged my fork according the github tutorial. sorry.
Now rebased it and removed unneccesary commits.

jan-cerny commented on linux_os/guide/system/permissions/partitions/mount_option_home_nodev/rule.yml:
```diff
@@ -28,9 +28,11 @@ identifiers:
     anssi: NT28(R12)
     srg: SRG-OS-000368-GPOS-00154
     cis@sle15: 1.1.15
+    cis@ubuntu1804: 1.1.13
```
this is twice here

willumpie replied:
Ay, duplicate. I will remove it

jan-cerny commented on linux_os/guide/system/permissions/partitions/mount_option_tmp_noexec/rule.yml:
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8,sle15
```
Why is sle15 removed?

willumpie replied:
It might be part of the merging and rebasing issues.
It was not my intention to remove it.
I will restore it.

jan-cerny commented on linux_os/guide/system/permissions/partitions/mount_option_tmp_noexec/rule.yml:
```diff
@@ -37,7 +37,6 @@ references:
     cis-csc: 11,13,14,3,8,9
     anssi: NT28(R12)
     srg: SRG-OS-000368-GPOS-00154
-    cis@sle15: 1.1.6
```
sle15 removed

willumpie replied:
restored

jan-cerny commented:
@openscap-ci ok to test

jan-cerny commented:
@openshift-ci-robot 

/ok-to-test

jan-cerny commented:
@willumpie Great! Thanks.

willumpie commented:
> jan-cerny: Thanks for merging.
